providers/database_provider.ts

```diff
@@ -80,6 +87,20 @@ export default class DatabaseServiceProvider {
     }
   }
 
+  /**
+   * Register TestUtils database macro
+   */
+  protected async registerTestUtils(db: Database) {
+    if (this.app.getEnvironment() === 'test') {
```

Comment:
not sure about this one. Naively, i would say no, but : could it cause some problems to register testUtils bindings only in test environnement ? 

Just wondering